### PR TITLE
NNS wrapper extensions

### PR DESCRIPTION
**Pull request by roman-khimov**
Inspired by https://github.com/nspcc-dev/neofs-node/pull/2841.

### CODE REVIEW

File: rpc/nns/hashes.go [hunk 1]
```diff
@@ -36,27 +36,69 @@ func InferHash(sg ContractStateGetter) (util.Uint160, error) {
...
+func NewFSReader(sg ContractStateGetter, invoker Invoker) (*ContractReader, error) {
+	h, err := InferHash(sg)
+	if err != nil {
+		return nil, err
+	}
+	return NewReader(invoker, h), nil
+}
+
+// NewFS creates an instance of [Contract] using hash obtained via [InferHash].
+func NewFS(sg ContractStateGetter, actor Actor) (*Contract, error) {
```
Comment (cthulhu-rider) [inline rpc/nns/hashes.go:50]: confusing name to me, i suggest
```suggestion
func NewInferred(sg ContractStateGetter, actor Actor) (*Contract, error) {
```
or
```suggestion
func NewWithInferredHash(sg ContractStateGetter, actor Actor) (*Contract, error) {
```

**Review by cthulhu-rider — COMMENTED** [review]
nice except some naming

File: rpc/nns/hashes.go (hunk 1 above)
Comment (roman-khimov) [reply rpc/nns/hashes.go:50]: Yeah, I had some doubts about it as well, but decided to try with a shorter one. @carpawell, what'd be your preference?

Comment (carpawell) [reply rpc/nns/hashes.go:50]: yes, i would die if i did not know what and why FS is here. NewInferred and NewWithInferredHash are not perfect to me either but better

Comment (roman-khimov) [reply rpc/nns/hashes.go]: Fixed.

**Review by carpawell — APPROVED** [review]

**Pull request merged**